<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="FileNotFoundException.xml" source-language="en-US" target-language="de-DE">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5e1b479aae634bc6fe5d96e28b51a1e223a82440c.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e1b479aae634bc6fe5d96e28b51a1e223a82440c</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.5,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.IO.FileNotFoundException">
          <source>The exception that is thrown when an attempt to access a file that does not exist on disk fails.</source>
          <target state="translated">Die Ausnahme, die bei dem Versuch ausgelöst wird, auf eine Datei zuzugreifen, die auf dem Laufwerk nicht vorhanden ist.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.FileNotFoundException">
          <source><ph id="ph1">`FileNotFoundException`</ph> uses the HRESULT COR_E_FILENOTFOUND which has the value 0x80070002.</source>
          <target state="translated"><ph id="ph1">`FileNotFoundException`</ph> verwendet die HRESULT COR_E_FILENOTFOUND mit dem Wert 0 x 80070002.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.FileNotFoundException">
          <source>If your code does not have <ph id="ph1">&lt;xref:System.Security.Permissions.FileIOPermissionAttribute.PathDiscovery%2A&gt;</ph> permission, the error message for this exception may only contain file or directory names instead of fully qualified paths.</source>
          <target state="translated">Wenn der Code nicht <ph id="ph1">&lt;xref:System.Security.Permissions.FileIOPermissionAttribute.PathDiscovery%2A&gt;</ph> Berechtigung, die Fehlermeldung für diese Ausnahme nur Datei oder das Verzeichnis Namen anstelle von voll qualifizierten Pfade enthalten kann.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="T:System.IO.FileNotFoundException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with optional message string and inner <ph id="ph2">&lt;see langword="Exception" /&gt;</ph> reference.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph>-Klasse mit einer optionalen Nachrichtenzeichenfolge und einem Verweis auf eine innere <ph id="ph2">&lt;see langword="Exception" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with its message string set to a system-supplied message.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph>-Klasse, bei der die Nachrichtenzeichenfolge auf eine Systemmeldung eingestellt wurde.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "Could not find the specified file."</source>
          <target state="translated">Dieser Konstruktor initialisiert die <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> -Eigenschaft der neuen Instanz mit einer vom System gelieferten Meldung, die den Fehler beschreibt, wie z. B. "wurde nicht gefunden die angegebene Datei."</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">Diese Meldung berücksichtigt die aktuelle Systemkultur.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>A description of the error.</source>
          <target state="translated">Eine Beschreibung des Fehlers.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>The content of <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> is intended to be understood by humans.</source>
          <target state="translated">Der Inhalt von <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> soll in verständlicher Sprache gehalten sein.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Der Aufrufer dieses Konstruktors muss sicherstellen, dass diese Zeichenfolge für die aktuelle Systemkultur lokalisiert wurde.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph>-Klasse mit einer angegebenen Fehlermeldung.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property of the new instance using <ph id="ph2">`message`</ph>.</source>
          <target state="translated">Dieser Konstruktor initialisiert die <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> -Eigenschaft der neuen Instanz unter Verwendung <ph id="ph2">`message`</ph>.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>An object that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Ein Objekt, das die serialisierten Objektdaten für die ausgelöste Ausnahme enthält.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>An object that contains contextual information about the source or destination.</source>
          <target state="translated">Ein Objekt, das die Kontextinformationen über die Quelle oder das Ziel enthält.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with the specified serialization and context information.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph>-Klasse mit der angegebenen Serialisierung und den angegebenen Kontextinformationen.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>A description of the error.</source>
          <target state="translated">Eine Beschreibung des Fehlers.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The content of <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> is intended to be understood by humans.</source>
          <target state="translated">Der Inhalt von <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> soll in verständlicher Sprache gehalten sein.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Der Aufrufer dieses Konstruktors muss sicherstellen, dass diese Zeichenfolge für die aktuelle Systemkultur lokalisiert wurde.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Die Ausnahme, die die Ursache der aktuellen Ausnahme ist.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Wenn der <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept>-Parameter nicht <ph id="ph1">&lt;see langword="null" /&gt;</ph> ist, wird die aktuelle Ausnahme in einem <ph id="ph2">&lt;see langword="catch" /&gt;</ph>-Block ausgelöst, der die innere Ausnahme behandelt.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph>-Klasse mit einer angegebenen Fehlermeldung und einem Verweis auf die innere Ausnahme, die diese Ausnahme ausgelöst hat.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Eine Ausnahme, die als direktes Ergebnis einer vorhergehenden Ausnahme ausgelöst wird, muss in der <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft über einen Verweis auf die vorhergehende Ausnahme verfügen.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Die <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft gibt den gleichen Wert zurück, der an den Konstruktor übergeben wird, oder <ph id="ph2">`null`</ph>, wenn die <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft den Wert der inneren Ausnahme nicht an den Konstruktor übergibt.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>.</source>
          <target state="translated">In der folgenden Tabelle werden die anfänglichen Eigenschaftenwerte für eine <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>-Instanz aufgeführt.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Eigenschaft</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Wert</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">Der Verweis auf die interne Ausnahme.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">Die Zeichenfolge der Fehlermeldung.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.Exception)">
          <source>For more information on inner exceptions, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Weitere Informationen zu inneren Ausnahmen finden Sie unter <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>A description of the error.</source>
          <target state="translated">Eine Beschreibung des Fehlers.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The content of <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> is intended to be understood by humans.</source>
          <target state="translated">Der Inhalt von <bpt id="p1">&lt;c&gt;</bpt>message<ept id="p1">&lt;/c&gt;</ept> soll in verständlicher Sprache gehalten sein.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Der Aufrufer dieses Konstruktors muss sicherstellen, dass diese Zeichenfolge für die aktuelle Systemkultur lokalisiert wurde.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The full name of the file with the invalid image.</source>
          <target state="translated">Der vollständige Name der Datei mit dem ungültigen Abbild.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message, and the file name that cannot be found.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph>-Klasse mit einer angegebenen Fehlermeldung und dem Dateinamen, der nicht gefunden werden kann.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String)">
          <source>The constructor initializes the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property of the new instance using <ph id="ph2">`message`</ph> and the <ph id="ph3">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> property using <ph id="ph4">`fileName`</ph>.</source>
          <target state="translated">Der Konstruktor initialisiert die <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> -Eigenschaft der neuen Instanz unter Verwendung <ph id="ph2">`message`</ph> und <ph id="ph3">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> Eigenschaft mit <ph id="ph4">`fileName`</ph>.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Die Fehlermeldung, in der die Ursache der Ausnahme erklärt wird.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The full name of the file with the invalid image.</source>
          <target state="translated">Der vollständige Name der Datei mit dem ungültigen Abbild.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Die Ausnahme, die die Ursache der aktuellen Ausnahme ist.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Wenn der <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept>-Parameter nicht <ph id="ph1">&lt;see langword="null" /&gt;</ph> ist, wird die aktuelle Ausnahme in einem <ph id="ph2">&lt;see langword="catch" /&gt;</ph>-Block ausgelöst, der die innere Ausnahme behandelt.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph> class with a specified error message, the file name that cannot be found, and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.IO.FileNotFoundException" /&gt;</ph>-Klasse mit einer angegebenen Fehlermeldung, dem Dateinamen, der nicht gefunden werden kann und einem Verweis auf die innere Ausnahme, die diese Ausnahme ausgelöst hat.</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Eine Ausnahme, die als direktes Ergebnis einer vorhergehenden Ausnahme ausgelöst wird, muss in der <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft über einen Verweis auf die vorhergehende Ausnahme verfügen.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">Die <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft gibt den gleichen Wert zurück, der an den Konstruktor übergeben wird, oder <ph id="ph2">`null`</ph>, wenn die <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph>-Eigenschaft den Wert der inneren Ausnahme nicht an den Konstruktor übergibt.</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>.</source>
          <target state="translated">In der folgenden Tabelle werden die anfänglichen Eigenschaftenwerte für eine <ph id="ph1">&lt;xref:System.IO.FileNotFoundException&gt;</ph>-Instanz aufgeführt.</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Eigenschaft</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Wert</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">Der Verweis auf die interne Ausnahme.</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">Die Zeichenfolge der Fehlermeldung.</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.#ctor(System.String,System.String,System.Exception)">
          <source>For more information on inner exceptions, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Weitere Informationen zu inneren Ausnahmen finden Sie unter <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.FileName">
          <source>Gets the name of the file that cannot be found.</source>
          <target state="translated">Ruft den Namen der Datei ab, die nicht gefunden werden kann.</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FileName">
          <source>The name of the file, or <ph id="ph1">&lt;see langword="null" /&gt;</ph> if no file name was passed to the constructor for this instance.</source>
          <target state="translated">Der Name der Datei oder <ph id="ph1">&lt;see langword="null" /&gt;</ph>, wenn kein Dateiname an den Konstruktor für diese Instanz übergeben wurde.</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FileName">
          <source>This property is read-only.</source>
          <target state="translated">Diese Eigenschaft ist schreibgeschützt.</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>Gets the log file that describes why loading of an assembly failed.</source>
          <target state="translated">Ruft die Protokolldatei ab, in der die Ursache für den Fehler beim Laden einer Assembly erläutert wird.</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>The errors reported by the assembly cache.</source>
          <target state="translated">Die vom Assemblycache gemeldeten Fehler.</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>The caller does not have the required permission.</source>
          <target state="translated">Der Aufrufer verfügt nicht über die erforderliche Berechtigung.</target>       </trans-unit>
        <trans-unit id="155" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>to provide evidence and view and modify policy.</source>
          <target state="translated">beweisen und zum Anzeigen und Ändern von Richtlinien.</target>       </trans-unit>
        <trans-unit id="156" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.FusionLog">
          <source>Associated enumerations: <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlEvidence" /&gt;</ph>, <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlPolicy" /&gt;</ph>.</source>
          <target state="translated">Zugeordnete Enumerationen: <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlEvidence" /&gt;</ph>, <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.ControlPolicy" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="157" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Das Objekt, das die serialisierten Objektdaten für die ausgelöste Ausnahme enthält.</target>       </trans-unit>
        <trans-unit id="158" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that contains contextual information about the source or destination.</source>
          <target state="translated">Das Objekt, das die Kontextinformationen für die Quelle oder das Ziel enthält.</target>       </trans-unit>
        <trans-unit id="159" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Sets the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object with the file name and additional exception information.</source>
          <target state="translated">Legt das <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>-Objekt mit dem Dateinamen und zusätzlichen Informationen zur Ausnahme fest.</target>       </trans-unit>
        <trans-unit id="160" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>for providing serialization services.</source>
          <target state="translated">für das Bereitstellen von Serialisierungsdiensten.</target>       </trans-unit>
        <trans-unit id="161" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Associated enumeration:   <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.SerializationFormatter" /&gt;</ph> Security action: <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityAction.LinkDemand" /&gt;</ph></source>
          <target state="translated">Zugeordnete Enumeration: <ph id="ph1">&lt;see cref="F:System.Security.Permissions.SecurityPermissionFlag.SerializationFormatter" /&gt;</ph> Sicherheitsaktion: <ph id="ph2">&lt;see cref="F:System.Security.Permissions.SecurityAction.LinkDemand" /&gt;</ph></target>       </trans-unit>
        <trans-unit id="162" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>requires full trust for the immediate caller.</source>
          <target state="translated">erfordert volle Vertrauenswürdigkeit für den unmittelbaren Aufrufer.</target>       </trans-unit>
        <trans-unit id="163" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This member cannot be used by partially trusted or transparent code.</source>
          <target state="translated">Dieser Member kann nicht von teilweise vertrauenswürdigem oder transparentem Code verwendet werden.</target>       </trans-unit>
        <trans-unit id="164" translate="yes" xml:space="preserve" uid="P:System.IO.FileNotFoundException.Message">
          <source>Gets the error message that explains the reason for the exception.</source>
          <target state="translated">Ruft die Fehlermeldung ab, in der die Ursache der Ausnahme erklärt wird.</target>       </trans-unit>
        <trans-unit id="165" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>The error message.</source>
          <target state="translated">Die Fehlermeldung.</target>       </trans-unit>
        <trans-unit id="166" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>If no message was supplied to the constructor for the current exception, this property returns a system-supplied error message.</source>
          <target state="translated">Wenn keine Meldung an den Konstruktor für die aktuelle Ausnahme bereitgestellt wurde, gibt diese Eigenschaft eine vom System bereitgestellten Fehlermeldung.</target>       </trans-unit>
        <trans-unit id="167" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>If the <ph id="ph1">&lt;xref:System.IO.FileLoadException.FileName%2A&gt;</ph> property is not <ph id="ph2">`null`</ph>, the message includes the file name.</source>
          <target state="translated">Wenn die <ph id="ph1">&lt;xref:System.IO.FileLoadException.FileName%2A&gt;</ph> Eigenschaft ist nicht <ph id="ph2">`null`</ph>, die Meldung enthält den Dateinamen.</target>       </trans-unit>
        <trans-unit id="168" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.IO.FileNotFoundException.Message">
          <source>This property is read-only.</source>
          <target state="translated">Diese Eigenschaft ist schreibgeschützt.</target>       </trans-unit>
        <trans-unit id="169" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.ToString">
          <source>Returns the fully qualified name of this exception and possibly the error message, the name of the inner exception, and the stack trace.</source>
          <target state="translated">Gibt den vollqualifizierten Namen dieser Ausnahme und ggf. die Fehlermeldung sowie den Namen der inneren Ausnahme und die Stapelüberwachung zurück.</target>       </trans-unit>
        <trans-unit id="170" translate="yes" xml:space="preserve" uid="M:System.IO.FileNotFoundException.ToString">
          <source>The fully qualified name of this exception and possibly the error message, the name of the inner exception, and the stack trace.</source>
          <target state="translated">Der vollqualifizierte Name dieser Ausnahme und ggf. die Fehlermeldung sowie der Name der inneren Ausnahme und die Stapelüberwachung.</target>       </trans-unit>
        <trans-unit id="171" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.ToString">
          <source>This method overrides <ph id="ph1">&lt;xref:System.Object.ToString%2A&gt;</ph>.</source>
          <target state="translated">Diese Methode überschreibt <ph id="ph1">&lt;xref:System.Object.ToString%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="172" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.ToString">
          <source>The string representation returned by this method includes the name of the exception, the value of the <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> property, the result of calling ToString on the inner exception, the value of the <ph id="ph2">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> property, and the result of calling <ph id="ph3">&lt;xref:System.Environment.StackTrace%2A&gt;</ph>.</source>
          <target state="translated">Die Zeichenfolgendarstellung von dieser Methode zurückgegebene enthält den Namen der Ausnahme, den Wert des der <ph id="ph1">&lt;xref:System.IO.FileNotFoundException.Message%2A&gt;</ph> -Eigenschaft, die das Ergebnis von Aufrufen von ToString für die innere Ausnahme, den Wert des der <ph id="ph2">&lt;xref:System.IO.FileNotFoundException.FileName%2A&gt;</ph> -Eigenschaft, und das Ergebnis des Aufrufs <ph id="ph3">&lt;xref:System.Environment.StackTrace%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="173" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.FileNotFoundException.ToString">
          <source>If any of these members is a null reference, its value is not included in the returned string.</source>
          <target state="translated">Wenn keines dieser Member ist ein null-Verweis, ist der Wert nicht in der zurückgegebenen Zeichenfolge enthalten.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>